README.md

```diff
@@ -1,3 +1,58 @@
 # Pantavisor
 
-The near bare metal multi-container supervisor and OS management system.
+## What is Pantavisor? 
+Pantavisor is the easiest way to build and manage embedded Linux projects with lightweight containers. Put your Linux distribution or custom-made firmware and userland into containers and get all of the benefits of portable containerized lifecycle management without needing to replace your distribution. 
```

Comment:
Here is like we have two definitions but we are not sure about which one to use. I'd prefer the second (because the first one is already explained in "Build your embedded system with containerized building blocks"

Reply:
I think I'll leave this one as is. Need to repeat these benefits near the top, otherwise people might not read down further.